{
    "repository": "caixw/gobuild",
    "clone_url": "https://github.com/caixw/gobuild.git",
    "timestamp": "2023-05-30T11:13:25.105945Z",
    "clone_success": true,
    "number of actions": 3,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_stdout": "[Test/Test] \ud83d\ude80  Start image=crawlergpt:latest\n[Test/Test]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[Test/Test]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Test/Test]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Test/Test]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /tmp/c66e4e10-fed3-11ed-a890-af2cc187fc11/caixw-gobuild] user=0 workdir=\n[Test/Test]   \u2601  git clone 'https://github.com/actions/setup-go' # ref=v2\n[Test/Test]   \u2601  git clone 'https://github.com/codecov/codecov-action' # ref=v1\n[Test/Test] \ud83e\uddea  Matrix: map[go:1.20.x os:ubuntu-latest]\n[Test/Test] \u2b50 Run Main Set git to use LF\n[Test/Test]   \ud83d\udc33  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/0] user= workdir=\n[Test/Test]   \u2705  Success - Main Set git to use LF\n[Test/Test] \u2b50 Run Main Check out code into the Go module directory\n[Test/Test]   \u2705  Success - Main Check out code into the Go module directory\n[Test/Test] \u2b50 Run Main Set up Go 1.20.x\n[Test/Test]   \ud83d\udc33  docker cp src=/home/andre-silva/.cache/act/actions-setup-go@v2/ dst=/var/run/act/actions/actions-setup-go@v2/\n[Test/Test]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /var/run/act/actions/actions-setup-go@v2/] user=0 workdir=\n[Test/Test]   \ud83d\udc33  docker exec cmd=[node /var/run/act/actions/actions-setup-go@v2/dist/index.js] user= workdir=\n[Test/Test]   \u274c  Failure - Main Set up Go 1.20.x\n[Test/Test] Error response from daemon: Could not find the file /var/run/act/workflow/pathcmd.txt in container 5c0435db0c50c13470139d9e9b10e43373b4b084ef69a1f1bfefb2b5a0ad26e8\n[Test/Test] failed to remove container: Error response from daemon: No such container: 5c0435db0c50c13470139d9e9b10e43373b4b084ef69a1f1bfefb2b5a0ad26e8\n[Test/Test] \ud83c\udfc1  Job failed\n",
    "actions_stderr": "Error: Error occurred running finally: Error occurred running finally: Error response from daemon: remove act-Test-Test-e9ad80ce940953c648e99da9e7b82d7b326237fe669778237ac8844052b293ad-env: volume is in use - [2b7bb31e22813a83864f4e48539c11ed9971955ed5dbad25d7be21ab8c4cc85d] (original error: <nil>) (original error: <nil>)\n"
}